Trigger/src/SConscript

```diff
@@ -67,6 +67,8 @@ helper.make_plugins( [ mainlib,
                        'xerces-c',
                        'boost_filesystem',
                        'hep_concurrency',
+                       'KinKal_Trajectory',
```

Comment:
I don't see other trigger related changes in this PR. Does this belong in this PR?